lab1-kots/lab1-e1-ui-analyzers/manifests/replicated-app.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: app-slug
+spec:
+  title: App Name
+  icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.png
```

Comment:
Should we change the default icon to something else? A new icon would cater to the Puppet training and differentiate this from our `starter-kots` repo.